crates/sui-types/src/base_types.rs

```diff
@@ -264,7 +264,7 @@ impl IntoPoint for TransactionDigest {
 pub struct ObjectDigest(
     #[schemars(with = "Base64")]
     #[serde_as(as = "Readable<Base64, Bytes>")]
-    pub [u8; 32],
+    pub [u8; TRANSACTION_DIGEST_LENGTH],
```

Comment:
this should have been Object Digest length right?

Reply:
Oof! Slipped through cracks.
Fix is here https://github.com/MystenLabs/sui/pull/4005